### DISCUSSION TITLE
New state for sleep - 2024.8.2

### DISCUSSION THREAD

**Discussion by tb205gti** [teslamate-org/teslamate#3780] [Q&A]
Have Tesla changed the sleep state in 2024.8.2? Since updating I get "offline" (state: 3) when the car is supposed to be in sleep.

Or does my car fail to send the "goodnight, I'm going to sleep" message to Tesla ever since updating to 2024.8.2?

**Comment by cwanja** [comment]
Please see #3753

**Reply by tb205gti** [reply]
Ahh, so they did change it 👍
